kiwi/tasks/base.py

```diff
@@ -127,9 +113,35 @@ def __init__(self, should_perform_task_setup: bool = True) -> None:
                     self.global_args['--logsocket']
                 )
 
+            # set log level
+            if self.global_args['--loglevel']:
+                try:
+                    log.setLogLevel(int(self.global_args['--loglevel']))
+                except ValueError:
+                    # Not a numeric log level, stick with the default
+                    # which is DEBUG
+                    pass
+            elif self.global_args['--debug']:
+                log.setLogLevel(logging.DEBUG)
+            else:
+                log.setLogLevel(logging.INFO)
```

Comment:
is this where the default logging level is set ? If so, shouldn't it be `DEBUG` as mentioned in line 122,
or update line 122 ?

Reply:
good point, line 122 is wrong :)